astropylibrarian/data/keywords.yaml

```diff
@@ -0,0 +1,93 @@
+astropy_package:
```

Comment:
Should we maybe sort these alphabetically? Or have core package stuff at top, affiliated below?

Reply:
Yeah, I've added an alphabetical sort via pre-commit.